### PR TITLE
Capstone project conference page

### PR DESCRIPTION
**Pull request by MudasirSherwani**
### Work Done on The Capston Project is mentioned below.

**1. Work Started with Mobile First Approach
2. Home Page
3. About Page
4. Project Mobile View
5. Project Desktop View
6. Featured Section Created Dynamically in Javascript
7. Mobile Hamburger Menu Created
8. Linters Added
9. Github Pages Deployment done
10. Intro Video Recorded
11. README File Update with link of Intro Video and Deployement**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,16 +4,148 @@
...
+            <span class="normal-font">2022.10.06</span>(THU) &
+            <span class="normal-font">08</span>(SAT)
+          </p>
+          <p class="program-location">@ Etihad Arena on Yas Island, Abu Dhabi</p>
+        </div>
+      </section>
     </main>
-    <footer class="footer">
-    </footer>
+              <!-- Program cards Section -->
```
Comment (alexander16108) [inline index.html:75]: Optional
on your hamburger menu, in the mobile version, you should change the pointer to notify users that it is clickable 😄

File: README.md [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
+## Intro Video Of Project
 
-![Project in Mobile View](images/11.png)!
+- [Watch Intro to NBA (The Capston Project)](https://www.loom.com/share/fe3063924366423ebb11204c5a07083d)
 
 ## Working Modules
 
 - Home Page
 - About Page
+- Mobile Hamburge Menu
```
Comment (alexander16108) [inline README.md:21]: The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/). 😄

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @MudasirSherwani , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!
Soo close you almost made it on your first try



## Highlights
- clean code ✔️ 
- passing linters ✔️ 
- passed requirements ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (MudasirSherwani) [reply index.html:75]: Add change cursor as a pointer on hover

File: README.md (hunk 2 above)
Comment (MudasirSherwani) [reply README.md:21]: README Acknowledgments section update with credit to Cindy Shin.

**Review by sumairq — APPROVED** [review]
Hi @MudasirSherwani  

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**